pennylane/tape/qnode.py

```diff
@@ -249,21 +257,39 @@ def _validate_backprop_method(device, interface):
         # determine if the device supports backpropagation
         backprop_interface = device.capabilities().get("passthru_interface", None)
 
+        # determine if the device has any child devices that support backpropagation
+        backprop_devices = device.capabilities().get("passthru_devices", None)

```

Comment:
Curious how this line wouldn't be covered by tests, when following ones are :thinking: Maybe it's a bug?
![image](https://user-images.githubusercontent.com/49409390/105103897-50549900-5a7f-11eb-9e5c-3abaa3a97fbf.png)

Reply:
Yeah, this doesn't seem possible 🤔